javalib/src/main/scala/java/util/concurrent/ConcurrentHashMap.scala

```diff
@@ -203,34 +203,62 @@ object ConcurrentHashMap {
    * hence the existence of this class, although it forwards all its operations
    * to the inner key set.
    */
-  class KeySetView[K, V] private[ConcurrentHashMap] (inner: Set[K])
+  class KeySetView[K, V] private[ConcurrentHashMap] (innerMap: ConcurrentHashMap[K, V], defaultValue: V)
```

Comment:
I think constructor requires 2nd parameter, because
* The instance method `ConcurrentHashMap#keySet()` passes `null.asInstanceOf[V]` to it
* The static method `newKeySet[K](): KeySetView[K, Boolean]` passes `true` to it

Reply:
Yes, that is correct. But why are you passing a `ConcurrentHashMap` as the first parameter and not a `InnerHashMap`? This seems unnecessary indirection.